ipn/localapi/localapi.go

```diff
@@ -158,16 +161,10 @@ type Handler struct {
 	// cert fetching access.
 	PermitCert bool
 
-	// CallerIsLocalAdmin is whether the this handler is being invoked as a
-	// result of a LocalAPI call from a user who is a local admin of the current
-	// machine.
-	//
-	// As of 2023-10-26 it is only populated on Windows.
-	//
-	// It can be used to to restrict some LocalAPI operations which should only
-	// be run by an admin and not unprivileged users in a computing environment
-	// managed by IT admins.
-	CallerIsLocalAdmin bool
+	// ConnIdentity of the client connected to the Handler.
+	ConnIdentity *ipnauth.ConnIdentity
+	// Test-only override for connIsLocalAdmin method.
```

Comment:
blank line before this

I assume "If non-nil, its pointed-to value is used." or something?